test/integration/patients/patient/patient.js

```diff
@@ -11,10 +11,20 @@ context('patient page', function() {
 
         return fx;
       })
-      .routePatientActions(_.identity, '1')
-      .routePatientFlows()
+      .routePatientActions(fx => {
+        fx.included = [];
```

Comment:
Since the context trail now re-renders more frequently, we needed to wait for these requests to complete before looking for the context trail element.

However, the data from these requests was overriding the test patient data. Since we aren't specifically testing anything related to actions/flows, emptying out the included data fixes that. But we are using it in a number of places, so perhaps a more universal fix is needed.

Reply:
Ok, dug into this again and realized this change is unnecessary. Just passing any id other than '1' as the second argument fixes the problem (I thought I had tried that earlier, but must have missed it). 

So the root of the issue is that `routePatientActions` and `routePatientFlows` default to assigning an id of '1' to all included `patients`. Might not be such a big issue since it's easy enough to pass a different id or use a different id in the test.